templates/auth_kerb.conf.erb

```diff
@@ -10,6 +10,17 @@
   KrbLocalUserMapping On
   # require valid-user
   require pam-account <%= scope.lookupvar('::foreman::pam_service') %>
+  # Set headers for proxy requests
+  RewriteRule .* - [E=HTTP_REMOTE_USER:%{REMOTE_USER}]
+  RequestHeader set REMOTE_USER %{REMOTE_USER}e
+  RewriteRule .* - [E=HTTP_REMOTE_USER_EMAIL:%{REMOTE_USER_EMAIL}]
+  RequestHeader set REMOTE_USER_EMAIL %{REMOTE_USER_EMAIL}e
+  RewriteRule .* - [E=HTTP_REMOTE_USER_FIRSTNAME:%{REMOTE_USER_FIRSTNAME}]
+  RequestHeader set REMOTE_USER_FIRSTNAME %{REMOTE_USER_FIRSTNAME}e
+  RewriteRule .* - [E=HTTP_REMOTE_USER_LASTNAME:%{REMOTE_USER_LASTNAME}]
+  RequestHeader set REMOTE_USER_LASTNAME %{REMOTE_USER_LASTNAME}e
+  RewriteRule .* - [E=HTTP_REMOTE_USER_GROUPS:%{REMOTE_USER_GROUPS}]
+  RequestHeader set REMOTE_USER_GROUPS %{REMOTE_USER_GROUPS}e
```

Comment:
Should each of these also come with a corresponding line to unset the value before Apache sets it?

Reply:
Since set replaces the existing value, the unset should not really be required. I have added unset anyways since I have seen other projects follow that practice sometimes.